hw/top_darjeeling/templates/toplevel.sv.tpl

```diff
@@ -396,6 +405,21 @@ for rst in output_rsts:
   assign lpg_rst_en[${k}] = ${rst_en};
 % endfor
 
+% for alert_group, lpgs in top['outgoing_alert_lpgs'].items():
+  // Outgoing LPGs for alert group ${alert_group}
+  % for k, lpg in enumerate(lpgs):
+  // ${lpg['name']}
+<%
+    cg_en = top['clocks'].hier_paths['lpg'] + lpg['clock_connection'].split('.clk_')[-1]
```

Comment:
That one is defined in [here](https://github.com/lowRISC/opentitan/pull/25179/files#diff-e4870f59f3d7b3080fabf9d5216c5a87104b49a1b98b1b9bb55b3157d1d4e612L679)

Reply:
Oh! I see: the definitions are things like `clkmgr_aon_clocks.clk_io_div4_peri` and this wants to get `io_div4_peri` here. (A singleton but I hadn't twigged we weren't splitting on spaces)

If I've got it right, this is essentially the `clk` signal in `merge.py`, line 900? Is there any way that the generated json file can preserve more of the structure? If not, maybe it makes sense to be more explicit about how we're splitting things here? Since this split happens several times in the code, maybe it needs to be in a helper function.

Reply:
In the last commit, I simplified that a lot I think.

Reply:
I really like it. Thanks!